Tools/clinic/clinic.py

```diff
@@ -1730,7 +1733,7 @@ def dump(self):
 # "languages" maps the name of the language ("C", "Python").
 # "extensions" maps the file extension ("c", "py").
 languages = { 'C': CLanguage, 'Python': PythonLanguage }
-extensions = { name: CLanguage for name in "c cc cpp cxx h hh hpp hxx".split() }
+extensions = { name: CLanguage for name in "c cc cpp cxx h hh hpp hxx test".split() }
```

Comment:
Forgot this as well

Reply:
Why this is added?

Reply:
I used this to regenerate the `clinic.test` file and forgot to remove it. It's now fixed.